libraries/references/schemas/metadata-metaschema.yml

```diff
@@ -27,9 +37,14 @@ allOf:
           version:
             title: "Version of the document format"
             type: integer
-          additionalProperties: false
+        additionalProperties: false
         required:
           - version
           - name
+    additionalProperties: true
     required:
       - metadata
+
+allOf:
+  - {$ref: "./metaschema.json#"}
```

Comment:
Is this definitely legal?
I think this might need to be fully qualified, like `$schema`. Then `$schema` tells us that this document conforms to our `metaschema.json`, and this `$ref` says that anything that conforms to this schema also conforms to `metaschema.json`. Would `$ref: "/schemas/common/metaschema.json#"` get expanded like `$schema` gets expanded on line 1, if we used that instead? Or does expansion only occur on the `$schema` property?

Reply:
(discussed in irc) yep, `$ref` can be relative, and that avoids the need to find and rewrite `$ref` entries when making absolute schemas.